docs/tutorials/TracingCode.md

```diff
@@ -177,7 +177,7 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
 
 1. Stepping through the method shows that it calls `ArgumentTokenizer#tokenize()` and `ParserUtil#parseIndex()` to obtain the arguments and index required.
 
-1. The rest of the method seems to exhaustively check for the existence of each possible parameter of the `edit` command and store any possible changes in an `EditPersonDescriptor`. Recall that we can verify the contents of `editPersonDesciptor` through the 'Variables' window.<br>
+1. The rest of the method seems to exhaustively check for the existence of each possible parameter of the `edit` command and store any possible changes in an `EditContactDescriptor`. Recall that we can verify the contents of `editContactDesciptor` through the 'Variables' window.<br>
```

Comment:
should `editContactDescriptor` be `EditContactDescriptor` also?

Reply:
followed convention